uiowa/uiowa
# Adding spellcheck for text fields/areas.

tracks #6972



# How to test
1. Sync facilities
2. Go to the alert content type
3. Enter incorrectly spelled text on textfields and textareas
4. Observe that they are spellchecked
5. profit.
